qiskit/pulse/schedule.py

```diff
@@ -62,7 +62,16 @@ def __init__(self, *schedules: List[Union[ScheduleComponent, Tuple[int, Schedule
         try:
             self._timeslots = TimeslotCollection(*timeslots)
         except PulseError as ts_err:
-            raise PulseError('Child schedules {0} overlap.'.format(schedules)) from ts_err
+            formatted_schedules = []
+            for sched_pair in schedules:
+                if not isinstance(sched_pair, (list, tuple)):
+                    sched_pair = (0, sched_pair)
+                _, sched = tuple(sched_pair)
+                sched = 'Schedule(name="{0}", duration={1})'.format(sched.name, sched.duration)
```

Comment:
It might be confusing if we use the same variable name for something of a different type. Can we call this `formatted_sched` or similar? I think it may also be easier to follow if you replace lines 67-69 with `sched = sched_pair[1] if isinstance(sched_pair, (List, tuple)) else sched_pair`

Reply:
fixed in https://github.com/Qiskit/qiskit-terra/pull/3319/commits/4543499f83421106d96dad2aff7a81602c64ee9b